account_payment_partner/wizard/payment_order_create.py

```diff
@@ -28,10 +28,10 @@ class PaymentOrderCreate(models.TransientModel):
 
     @api.model
     def extend_payment_order_domain(self, payment_order, domain):
-        res = super(PaymentOrderCreate, self).extend_payment_order_domain(
+        super(PaymentOrderCreate, self).extend_payment_order_domain(
             payment_order, domain)
         domain += ['|', '|',
                    ('invoice', '=', False),
                    ('invoice.payment_mode_id', '=', False),
                    ('invoice.payment_mode_id', '=', payment_order.mode.id)]
-        return res
+        return True
```

Comment:
Hum in fact many methods used to return True to avoid returning nothing and not be incompatible with the XMLRPC protocol, so may be this is the kind of dummy meaningless return True. Let's say I just fail to see how it's better to hardcode True than returning super. But anyway not blocking for such a detail.